src/dune/context.mli

```diff
@@ -51,12 +51,12 @@ type t = private
         (** [Some path/to/foo.exe] if this contexts is for feedback-directed
             optimization of target path/to/foo.exe *)
   ; fdo_target_exe : Path.t option
-        (* By default Dune builds and installs dynamically linked foreign
-           archives (usually named [dll*.so]). It is possible to disable this by
-           adding (disable_dynamically_linked_foreign_archives true) to the
-           workspace file, in which case bytecode executables will be built with
-           all foreign archives statically linked into the runtime system. *)
-  ; disable_dynamically_linked_foreign_archives : bool
+        (** By default Dune builds and installs dynamically linked foreign
+            archives (usually named [dll*.so]). It is possible to disable this
+            by adding (dynamically_linked_foreign_archives true) to the
```

Comment:
You modified the comment here, but haven't allowed the new syntax in the workspace file.

Reply:
Fixed